PROJ-175 using mock data (#103)
SW805F20/report: sections/sprint-3/mock-data.tex

Throughout the development of the game, the ability to continuously test new changes was needed.
Because of the lockdown, it became difficult to do testing with the Pozyx hardware and therefore another solution was required.
The solution was to create some mock data that could be used to get the field, the players, and the ball to be rendered on the screen.
The mock data is stored on the host side and then sent to the clients to test the formatting of messages and the overall flow of the game.
When using mock data, a small function was created to get a new random position for each of the players and the ball to allow for simulation of movement.

> The function returned the the players' or ball's current position plus some random step size in any random direction.

Reviewer: ```suggestion
The function returned the the players' or ball's current position plus a step of random size in a random direction.
```
